docker-compose.yaml

```diff
@@ -0,0 +1,49 @@
+services:
+  api:
+    image: pramsey/pg_featureserv:20240305
+    ports:
+      - "0.0.0.0:${OSM_PG_FEATURESERV_API_SYSTEM_PORT}:9000"
+    environment:
+      - DATABASE_URL=postgres://${OSM_PG_FEATURESERV_POSTGRES_USER}:${OSM_PG_FEATURESERV_POSTGRES_PASS}@db/${OSM_PG_FEATURESERV_POSTGRES_DBNAME}
+    depends_on:
+      db:
+        condition: service_healthy
+  db:
+    image: postgis/postgis:16-3.4
```

Comment:
the official postgis docker image automatically installs the tiger extension (see https://github.com/postgis/docker-postgis/issues/187), which we do not need at all.
that's why I originally used the kartoza postgis image. 
if we want to stick to the officical postgis image, we must mount a sql file with the content `create extension postgis` to the start directory

Reply:
I created a PR on this PR that fixes this: https://github.com/JakobMiksch/osm_feature_info/pull/11